pinconfigs/README.md

```diff
@@ -0,0 +1,148 @@
+# GD32 pin configurations
+
+This directory contains a set of files describing valid pin configurations for
+multiple GD32 devices, for example, that `USART0_TX` signal can only be mapped
+to `PA0` and `PC8`. These configurations can be used to generate valid pin
+mappings.
+
+GD32 devices use two distinct models to configure signal multiplexing. In
+some devices, like GD32F103XX, each pin has one or more associated alternate
+function. These alternate functions can be sometimes changed using the remap
+functionality, which allows routing all signals from a peripheral to a
+different set of pins. For this reason we will name the model used by such
+devices *AFIO model*. Some other devices, like GD32F450XX, have a simpler
+and more advanced model where each pin has a fixed list of alternate functions.
+Similarly, we will name the model used by such devices *AF model*.
```

Comment:
I'm missing info about how to select between AFIO vs AF models.
Is it possible select one or another to describe?